packages/worker/fileWorker.js

```diff
@@ -0,0 +1,186 @@
+//// TODO: Use power queue to handle throttling etc.
+//// Use observe to monitor changes and have it create tasks for the power queue
+//// to perform.
+
+/**
+ * @public
+ * @type Object
+ */
+FS.FileWorker = {};
+
+/**
+ * @method FS.FileWorker.observe
+ * @public
+ * @param {FS.Collection} fsCollection
+ * @returns {undefined}
+ *
+ * Sets up observes on the fsCollection to store file copies and delete
+ * temp files at the appropriate times.
+ */
+FS.FileWorker.observe = function(fsCollection) {
+
+  // Initiate observe for finding newly uploaded/added files that need to be stored
+  // per store.
+  FS.Utility.each(fsCollection.options.stores, function(store) {
+    var storeName = store.name;
+    fsCollection.files.find(getReadyQuery(storeName), {
+      fields: {
+        copies: 0
+      }
+    }).observe({
+      added: function(fsFile) {
+        // added will catch fresh files
+        FS.debug && console.log("FileWorker ADDED - calling saveCopy", storeName, "for", fsFile._id);
+        saveCopy(fsFile, storeName);
+      },
+      changed: function(fsFile) {
+        // changed will catch failures and retry them
+        FS.debug && console.log("FileWorker CHANGED - calling saveCopy", storeName, "for", fsFile._id);
+        saveCopy(fsFile, storeName);
+      }
+    });
+  });
+
+  // Initiate observe for finding files that have been stored so we can delete
+  // any temp files
+  fsCollection.files.find(getDoneQuery(fsCollection.options.stores)).observe({
+    added: function(fsFile) {
+      FS.debug && console.log("FileWorker ADDED - calling deleteChunks for", fsFile._id);
+      try {
+        FS.TempStore.removeFile(fsFile);
+      }catch(err) {
```

Comment:
Reaction coding style prefers to have spaces after the brackets } and before the parens
e.g.
`} catch (err) {`